tests/e2e/etcd_mix_versions_test.go

```diff
@@ -94,6 +94,7 @@ func mixVersionsSnapshotTest(t *testing.T, clusterVersion, newInstanceVersion e2
 	// start a new etcd instance, which will receive a snapshot from the leader.
 	newCfg := *epc.Cfg
 	newCfg.Version = newInstanceVersion
+	newCfg.SnapshotCatchUpEntries = 10
```

Comment:
It can work, but I suggest to make change something like below,

```
$ git diff  tests/e2e/etcd_mix_versions_test.go
diff --git a/tests/e2e/etcd_mix_versions_test.go b/tests/e2e/etcd_mix_versions_test.go
index ae11db6b1..3f3795810 100644
--- a/tests/e2e/etcd_mix_versions_test.go
+++ b/tests/e2e/etcd_mix_versions_test.go
@@ -70,6 +70,7 @@ func mixVersionsSnapshotTest(t *testing.T, clusterVersion, newInstanceVersion e2
        epc, err := e2e.NewEtcdProcessCluster(context.TODO(), t,
                e2e.WithClusterSize(1),
                e2e.WithSnapshotCount(10),
+               e2e.WithSnapshotCatchUpEntries(10),
                e2e.WithVersion(clusterVersion),
        )
        if err != nil {
```

Reply:
here test logic  is need to add and start second member, not create e new cluster,  I think we should still use `StartNewProc` not `NewEtcdProcessCluster ` here,   please point it out if I miss something  ?

Reply:
YES